smwwims.md

```diff
@@ -0,0 +1,40 @@
+---
+name: "@smwwims"
+project: Autonomous Plant Farm
+---
+
+# Autonomous Plant Farm
+
+## Summary
+
+I have always had a keen interest in botany, but unfortunately my gardening skills do not quite match my programming skills. 
+That is why I came up with the idea of building a fully automatic cultivation machine. In contrast to the "Smart Planting Sets" 
+available on the market, my machine is not a simple control circuit, but a regulation circuit. This means that the water and light 
+supply is not simply controlled by the time but responds to the needs of the plants via sensors.
+
+## Plan
+
+1. Preparing the plant pot:   Install the water tank, demarcate the electronic part
+2. Water supply:              Install pump, install water pipe
+3. Sensors:                   Wire the sensors
+4. Light:                     Install and wire the plant light
+5. Programming:               Make it all work through software
+
+## Budget
+
+| Product                     | Supplier/Link                                                                                | Cost   |
+| --------------------------- | -------------------------------------------------------------------------------------------- | ------ |
+| Plant Pot                   | https://www.amazon.de/-/en/gp/product/B08KWPGMGY/ref=ewc_pr_img_13?smid=A2WPX7PK44TEBQ&th=1  | $41.69 |
+| Water Tank                  | https://www.amazon.de/-/en/gp/product/B0B8YSNG5C/ref=ewc_pr_img_1?smid=A3TW56H8H7ZPJ&th=1    | $26.78 |
+| Aluminium Sheet             | https://www.amazon.de/-/en/gp/product/B07XC24C9G/ref=ewc_pr_img_1?smid=A34I19W3628WJL&th=1   | $20.41 |
+| Water Hose                  | https://www.amazon.de/-/en/gp/product/B07RDJY6DT/ref=ewc_pr_img_1?smid=A3JWKAKR8XB7XF&th=1   | $12.85 |
+| Plant Lamp                  | https://www.amazon.de/-/en/gp/product/B0B1MRYNWR/ref=ewc_pr_img_1?smid=A1Y73AAMUAXH83&th=1   | $25.71 |
+| Arduino Plant Watering Kit  | https://www.amazon.de/-/en/gp/product/B0BJ7W59SS/ref=ewc_pr_img_1?smid=A1X7QLRQH87QA3&th=1   | $25.92 |
+| LCD Display                 | https://www.amazon.de/-/en/gp/product/B07Y4RFLZ5/ref=ewc_pr_img_1?smid=A1A7E5ILEFA1R3&psc=1  | $15.64 |
+| Breadboard PCB              | https://www.amazon.de/-/en/gp/product/B0734XYJPM/ref=ewc_pr_img_1?smid=AZF7WYXU5ZANW&psc=1   | $14.99 |
```

Comment:
do you need wires? also, it looks like this protoboard is for soldering? do you already have a soldering set?

Reply:
Yes, I already have a soldering set and the wires. Should I add them to the list for clarity?

Reply:
yes please!